05-geometry-operations.qmd

```diff
@@ -268,9 +269,102 @@ Imagine you want to select not one circle or the other, but the space covered by
 x.intersection(y)
 ```
 
+The subsequent code chunks demonstrate how this works for the other combinations of the 'Venn' diagram representing x and y, inspired by Figure 5.1 of the book R for Data Science (Grolemund and Wickham 2016). In addition to `.intersection`, we hereby use the `.difference`, `.union`, and `.symmetric_difference` operators:
```

Comment:
I suggest:

```
The next lines of code demonstrate how this works for the `.difference`, `.union`, and `.symmetric_difference` operators:
```

Shorter and simpler and no reference to an R book which may seem out of place.

Reply:
Agree, thanks!